dist-persist/wbstack/src/Internal/ApiWbStackElasticSearchInit.php

```diff
@@ -14,8 +16,9 @@ public function execute() {
 
         @set_time_limit( 60*5 ); // 5 mins maybe D:
 		@ini_set( 'memory_limit', '-1' ); // also try to disable the memory limit? Is this even a good idea?
-		
-        $parameters = '--cluster all';
+
+		$cluster = $this->getParameter( 'cluster' );
```

Comment:
comment (i.e. non-blocking): this now means that we are deviating from our previous path of controlling which clusters are written in which way by only using the environment variables set at runtime on the mediawiki containers. A possible alternative to this would be to continue the pattern we have followed in the past of using kubernetes jobs (rather than making api calls to the shared backend mediawiki deployment).

Reply:
makes sense; I can also see us considering removing this even if we sunset the non-shared indices so that we manage to keep moving down the direction of "fewer jobs being run by api calls to the backend cluster"